mobile/library/cc/engine_builder.cc

```diff
@@ -864,8 +869,12 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
     auto* quic_protocol_options = alpn_options.mutable_auto_config()
                                       ->mutable_http3_protocol_options()
                                       ->mutable_quic_protocol_options();
-    quic_protocol_options->set_connection_options(http3_connection_options_);
-    quic_protocol_options->set_client_connection_options(http3_client_connection_options_);
+    if (!quic_connection_options_.empty()) {
+      quic_protocol_options->set_connection_options(absl::StrJoin(quic_connection_options_, ","));
+    } else {
+      quic_protocol_options->set_connection_options(http3_connection_options_);
+      quic_protocol_options->set_client_connection_options(http3_client_connection_options_);
```

Comment:
OOC, are we intentionally getting rid of client connection options? I believe we use both in QUICHE.

Reply:
Thanks for calling this out!  I added a method to also add client connection options.